TelCo.ColorCoder/PairsDataModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+	public class PairsDataModel

```

Comment:
`PairsDataModel` - good name :)

Reply:
Sir,
I added the manual.
I had a doubt regarding GetPairs.cs and GetColors.cs, they are doing very similar functions. Would it be better if we put these 2 in the same class? I was not able to do so because of the word limit.

Reply:
I didn't see much similarity in the code. Maybe what you mean is that they are related functionality... E.g., if we add colors, we will need to change both files.
It's also possible they will evolve differently, e.g. the error handling when a number is not found may be different than when a color is invalid.